app/services/hyrax/multiple_membership_checker.rb

```diff
@@ -20,53 +19,63 @@ def initialize(item:)
     # `allow_multiple_membership` as `false` require that its members do not
     # also belong to other collections of the same type.
     #
-    # There are two contexts in which memberships are checked: when doing a
-    # wholesale replacement and when making an incremental change, such as
-    # adding a single collection membership to an object. In the former case,
-    # `#check` only scans the passed-in collection identifiers. In the latter,
-    # `#check` must also scan the collections to which an object currently
-    # belongs for potential conflicts.
-    #
     # @param collection_ids [Array<String>] a list of collection identifiers
-    # @param include_current_members [Boolean] a flag to also scan an object's
-    #   current collection memberships
+    # @param include_current_members [Boolean] if true, include item's existing collections in check; else if false, check passed in collections only
```

Comment:
can we shorten the line lengths here? i try to set a soft cap at 80 chars especially in consideration of https://github.com/samvera-labs/bixby/issues/48.

since these lines can be wrapped without syntax considerations, it seems best to do so.

Reply:
I shortened all long lines.